Libplanet/Action/ActionEvaluator.cs

```diff
@@ -228,18 +197,21 @@ internal static ValidatorSet NullValidatorSetGetter()
 
         /// <summary>
         /// Retrieves the set of <see cref="Address"/>es that will be updated when
-        /// a given <see cref="Transaction{T}"/> is evaluated.
+        /// a given <see cref="ITransaction"/> is evaluated.
         /// </summary>
-        /// <param name="tx">The <see cref="Transaction{T}"/> to evaluate.</param>
+        /// <param name="tx">The <see cref="ITransaction"/> to evaluate.</param>
```

Comment:
The `tx` parameter seems like a `Transaction<T>` type yet. Is this okay?

Reply:
For `GetUpadatedAddresses()`, we still need to keep the type parameter since it needs materialization statically. but this behavior doesn't compatible with the action type loader concept and we need to remove it in near future. cc @planetarium/libplanet

in this PR, I'll fix the docstring and add comments about that.